### PR TITLE
[fix](paimon)Remove the static attribute of the source for paimon

### PR DESCRIPTION
**Pull request by wuwenchi**
## Proposed changes

Each `ScanNode` corresponds to its own source, so delete the `static` attribute.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by wuwenchi** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/planner/external/FileQueryScanNode.java [hunk 1]
```diff
@@ -260,6 +254,9 @@ public TFileScanRangeParams getFileScanRangeParams() {
         return params;
     }
 
+    protected void setScanParams(TFileRangeDesc rangeDesc, Split split) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/planner/external/FileQueryScanNode.java:257]: Add some comment for this method

**Comment by wuwenchi** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**